src/lightning/pytorch/strategies/launchers/xla.py

```diff
@@ -69,15 +68,28 @@ def launch(self, function: Callable, *args: Any, trainer: Optional["pl.Trainer"]
             **kwargs: Optional keyword arguments to be passed to the given function.
         """
         context = mp.get_context(self._start_method)
-        return_queue = context.SimpleQueue()
+
+        from torch_xla.experimental import pjrt
+
+        using_pjrt = pjrt.using_pjrt()
+        # pjrt requires that the queue is serializable, `SimpleQueue` is not
+        return_queue = context.Queue() if using_pjrt else context.SimpleQueue()
```

Comment:
Have you test the `mp.Queue`? I thought it is not serializable just like `mp.SimpleQueue`.

Reply:
You are right. Let me revert that